substrate/frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -108,6 +108,27 @@ impl From<FeasibilityError> for MinerError {
 	}
 }
 
+/// Reports the trimming result of mined solution
+#[derive(Debug, Clone)]
+pub struct TrimmingStatus {
```

Comment:
not sure about naming this, could also be `TrimmingResult` or something.

could also just be a boolean but I prefer a struct here